licenses/LICENSE.blockingconcurrentqueue.txt

```diff
@@ -0,0 +1,26 @@
+©2015-2016 Cameron Desrochers. Distributed under the terms of the simplified
+BSD license, available at the top of concurrentqueue.h.
+
+Uses Jeff Preshing's semaphore implementation (under the terms of its
+separate zlib license, embedded below).
+
+
+zlib license
```

Comment:
how do we avoid including this in source distribution?

Reply:
This is part of TVM. This is the same way they include the license for this file.

Reply:
See https://github.com/apache/tvm/blob/main/LICENSE#L223-L226 and https://github.com/apache/tvm/blob/main/licenses/LICENSE.blockingconcurrentqueue.txt for references in TVM.